components/list/list-item-button-mixin.js

```diff
@@ -96,6 +104,7 @@ export const ListItemButtonMixin = superclass => class extends ListItemMixin(sup
 	_renderPrimaryAction(labelledBy, content) {
 		return html`<button 
 			id="${this._primaryActionId}" 
+			aria-current="${ifDefined(this._ariaCurrent)}"
```

Comment:
Let me know thoughts on setting `aria-current` in button's render. It could also be a public property, but I left it private for now.

Reply:
Yeah I think this is the better way. Let's keep it private for now.